src/utils/openalex.py

```diff
@@ -44,51 +44,38 @@ def get_data_from_doi(self, doi):
             raise DOINotFoundError(f"No OpenAlex works found for doi: {doi}")
         return results[0]
 
-    def get_data_from_source(self, source, date, page=1):
```

Comment:
removing this caused an error here: https://quantfive.sentry.io/issues/4464676335/?referrer=alert_email&alert_type=email&alert_timestamp=1694206809609&alert_rule_id=885988&notification_uuid=15d9afb5-c33c-4bb1-a19a-1869125cccfc&environment=staging

Reply:
I'll check this out, might have removed it as part of the rebase

Reply:
I think it is good to keep these separate, at least for the time being. Concept and hub have overlap but are different enough where it makes sense to segregate the two.

For reference, we have the concept table

![image](https://github.com/ResearchHub/researchhub-backend/assets/802819/f696d400-8be0-4115-9d41-9c756d3b1cf4)

And the unified doc concept table created through the oneToOne relationship
![image](https://github.com/ResearchHub/researchhub-backend/assets/802819/7c502d46-0f72-4059-87a1-6f2ebbed1e1e)